.github/workflows/build-test.yml

```diff
@@ -136,6 +136,49 @@ jobs:
         command: make
         args: ci-job-test-docs
 
+  # Runs full datagen on prod data
+  full-datagen:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Load the default Rust toolchain via the rust-toolchain file.
+      run: rustup show
+    - name: Attempt to load cached source data
+      uses: actions/cache@v2
+      id: source-data-cache
+      with:
+        path: data/source
+        key: source-data-41-71
+    - name: Download CLDR source data
+      if: steps.source-data-cache.outputs.cache-hit != 'true'
+      uses: robinraju/release-downloader@v1.3
+      with: 
+        repository: "unicode-org/cldr-json"
+        tag: "41.0.0"
+        fileName: "cldr-41.0.0-json-full.zip"
+        out-file-path: "data/source/"
```

Comment:
Nit: Put the data in a path that is covered by `.gitignore` (or add this directory to `.gitignore`)

Reply:
Are these tests not hermetic?

Reply:
I just find it messy for generated files to go into the non-ignored area of a git repo; I think of a git repo as something that is readonly except within gitignore-covered directories

Reply:
I'd rather place it outside the repo, but the cache action doesn't seem to support that. I don't think we should add this to `.gitignore` though as it never runs locally and on CI it gets cleaned up immediately